### PR TITLE
[IMP] inventory: update one step shipments doc

### PR DESCRIPTION
**Pull request by brse-odoo**
This PR is an [IMP] targeting the `receipts_delivery_one_step` doc in Odoo's Inventory app documentation, specific to 17.0.

Here's what I did:
- Rewrote/updated copy and sections for accuracy specific to 17.0
- Replaced screenshots/added new screenshots with 17.0 UI
- Added admonitions and expanded doc to be more robust

[Project Task Link](https://www.odoo.com/web#cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form&id=3906627)

### CODE REVIEW

**Comment by brse-odoo** [comment]
@larm-odoo This PR is ready for content review, thank you in advance!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst:39]: I am a little confused- when you say the "above sttings" - I suggest calling out the setting specifically.

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst
Comment (larm-odoo) [inline content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst:41]: I am not sure if this is necessary in this doc? Or instea you can link to the other docs- the 2 and 3 step docs?

**Review by larm-odoo — APPROVED** [review]
Hi Brandon- nice doc, easy and understandable. I don't have a lot of suggestions- just two, really. I am approving now!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst:72]: I haven't been assigning it a guilabel since it's not actually written on the screen. But if I get this feedback again, I will implement it accordingly!

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst:39]: Nice call - Suggestion implemented ✅

File: content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst
Comment (brse-odoo) [reply content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_delivery_one_step.rst:41]: I understand the logic here, but because I show the shipments settings in the included screenshot, I think it's helpful to mention that this is required for that to show up. But again, if I get similar feedback, I'll adjust accordingly!

**Comment by brse-odoo** [comment]
@odoo/us-doc-review This PR is ready for final content review, thank you!

**Review by samueljlieber — CHANGES_REQUESTED** [review]
Hi @brse-odoo, everything looks good– however, please resolve the build errors before I approve? Thank you 🙏

**Comment by brse-odoo** [comment]
> Hi @brse-odoo, everything looks good– however, please resolve the build errors before I approve? Thank you 🙏

Hey @samueljlieber, sorry about that - all good to go now 👍

**Review by samueljlieber — APPROVED** [review]
Thank you @brse-odoo, your improvements in this PR look good to me! Nice work 👍
...
@robodoo r+

**Pull request closed without merging**
